src/en/vyvymanga/src/eu/kanade/tachiyomi/extension/en/vyvymanga/VyvyManga.kt

```diff
@@ -18,7 +18,7 @@ import java.util.Locale
 class VyvyManga : ParsedHttpSource() {
     override val name = "VyvyManga"
 
-    override val baseUrl = "https://vyvymanga.net"
+    override val baseUrl = "https://vymanga.net"
```

Comment:
`https://vyvymanga.net/` seems to still be up?

Reply:
They claim to be moving